### PR TITLE
Added focus groups

### PR DESCRIPTION
**Pull request by Nyholm**
FYI @wouterj

### CODE REVIEW

File: templates/startpage.html.twig [hunk 1]
```diff
@@ -42,11 +42,102 @@
...
+            <p>
+                One thing that is super helpful for maintainers is reproducing bugs.
+                It is way easier to resolve a bug report with a code example and
+                that is verified by someone.
+            </p>
+
+            <p>
+                Have a look at the <a href="https://github.com/symfony/symfony/issues?q=is%3Aissue+is%3Aopen+label%3ABug">open bug reports</a>,
+                try to understand and reproduce the bugs. Make a comment like
+                "Verified" or "Cannot reproduce". Also explain what you have tried,
```
Comment (wouterj) [inline templates/startpage.html.twig:84]:
```suggestion
                "Status: Reviewed" (if you can reproduce) or "Status: Works for me". Also explain what you have tried,
```

This way, Carson will correctly adjust the label.

File: templates/startpage.html.twig [hunk 2]
```diff
@@ -42,11 +42,102 @@
...
+            </p>
+
+            <p>
+                If the bug was verified and you want to walk the extra mile,
+                you can also try fixing it. Ask in Slack for help.
+            </p>
+
+            <h4>Review open PRs</h4>
+            <p>
+                If you really want a challenge, go the the <a href="https://github.com/symfony/symfony/pulls?page=2&q=is%3Apr+is%3Aopen+label%3A%22Status%3A+Needs+Review%22">open PRs</a>
```
Comment (wouterj) [inline templates/startpage.html.twig:95]:
```suggestion
                If you really want a challenge, go through the <a href="https://github.com/symfony/symfony/pulls?page=2&q=is%3Apr+is%3Aopen+label%3A%22Status%3A+Needs+Review%22">open PRs</a>
```

Not sure, but at least `the the` was probably not intended :)

File: templates/startpage.html.twig [hunk 3]
```diff
@@ -42,11 +42,102 @@
...
+                and review them. Make a comment about what you like and what could
+                be improved. Maybe event test the PR to make sure it works. Then try
+                to figure out why the PR is still open.
```
Comment (wouterj) [inline templates/startpage.html.twig:98]: Does it make sense to add a link to https://symfony.com/doc/current/contributing/community/reviews.html here?

File: templates/startpage.html.twig (hunk 1 above)
Comment (wouterj) [reply templates/startpage.html.twig:84]: Maybe also link to https://symfony.com/doc/current/contributing/community/reviews.html#the-bug-report-review-process here

File: templates/startpage.html.twig [hunk 4]
```diff
@@ -42,11 +42,102 @@
...
+                <li>Does it need a decision for the Core Team?</li>
+                <li>Does it need more work? If the PR has been stalled for a long time, maybe you want to finish it?</li>
+                <li>If it is not clear why it is stalled, make sure to submit a Github review with "Approve" or "Request changes"</li>
+            </ul>
+
+            <p>How to checkout someone else's PR?</p>
+            <code><pre>
+git fetch origin pull/38920/head:pr-38920
+git checkout pr-38920
+            </pre></code>
```
Comment (wouterj) [inline templates/startpage.html.twig:110]:
```suggestion
            <p>After <a href="https://symfony.com/doc/current/contributing/code/pull_requests.html#step-2-setup-your-environment">setting up</a> Symfony on your PC. You can use the following git commands to checkout a PR (in this case number 38920) locally:</p>
            <code><pre>
git fetch upstream pull/38920/head:pr-38920
git checkout pr-38920
            </pre></code>
```

File: templates/startpage.html.twig [hunk 5]
```diff
@@ -42,11 +42,102 @@
...
+
+            <h4>Tell us about testing</h4>
+            <p>
+                TODO
+            </p>
+            <h4>Write a blog post</h4>
+            <p>
+                An absolutely awesome way to contribute to symfony is to write a
```
Comment (wouterj) [inline templates/startpage.html.twig:118]:
```suggestion
                An absolutely awesome way to contribute to Symfony is to write a
```

File: templates/startpage.html.twig [hunk 6]
```diff
@@ -42,11 +42,102 @@
...
+                blog post where you share an idea or describe how you solved a problem.
+                Even if a problem is very specific, it will still be helpful for
+                some people.
+            </p>
+            <p>
+                Blog posts differs from the Symfony documentation because they can
+                be opinionated. There are sometimes mor than one way to solve a
```
Comment (wouterj) [inline templates/startpage.html.twig:125]:
```suggestion
                be opinionated. There are sometimes more than one way to solve a
```

File: templates/startpage.html.twig [hunk 7]
```diff
@@ -42,11 +42,102 @@
...
+                problem and none is "the best". The documentation is also very restrictive
+                what libraries to recommend.
+            </p>
+            <p>
+                If you write a blog post, make sure to spread it on Twitter, in the
+                #posts channel on Symfony Devs slack and on <a href="https://dev.to/">dev.to</a>.
```
Comment (wouterj) [inline templates/startpage.html.twig:131]: I won't add "dev.to" here. Imho, it's completely up to the writer were to post it (e.g. Medium is just as fine as dev.to, only in #posts and twitter also).

In fact, I think "make sure to share it in the #posts channel on Symfony slack and any other social media." would be better, don't you think?

**Review by wouterj — COMMENTED** [review]
Good work!
I've proposed some more doc references/fixed a few typos. I also think we should link to https://symfony.com/slack somewhere.

**Comment by Nyholm** [comment]
Thank you. 

We do link to symfony.com/slack. I snuck that in yesterday: https://github.com/Nyholm/sfhackday.com/pull/24/files#diff-62b500f7af68bca747c87be8d9a9c6948ef5fb08128e03e8c7eff9beb8d35cbfR40

File: templates/startpage.html.twig (hunk 1 above)
Comment (Nyholm) [reply templates/startpage.html.twig]: Excellent idea

File: templates/startpage.html.twig (hunk 3 above)
Comment (Nyholm) [reply templates/startpage.html.twig:98]: Sure!

File: templates/startpage.html.twig (hunk 7 above)
Comment (Nyholm) [reply templates/startpage.html.twig:132]: Sure.

**Review by wouterj — APPROVED** [review]
Ready imho

**Comment by Nyholm** [comment]
Awesome. Lets deploy!

**Pull request merged**